common/djangoapps/student/models.py

```diff
@@ -1595,6 +1597,12 @@ def emit_event(self, event_name, enterprise_uuid=None):
             segment_traits['email'] = self.user.email
 
             if event_name == EVENT_NAME_ENROLLMENT_ACTIVATED:
+                if should_send_redesign_email():
+                    course_pacing_type = 'self-paced' if self.course_overview.self_paced else 'instructor-paced'
```

Comment:
what does `self.mode` return?

Reply:
course mode

Reply:
I meant what is the value returned by it? Can we use that directly in our template instead of creating the value here